src/ParquetMsg.chpl

```diff
@@ -285,13 +301,20 @@ module ParquetMsg {
     var fileErrorMsg:string = "";
     var sizes: [filedom] int;
     var types: [dsetdom] ArrowTypes;
+    var byteSizes: [filedom] int;
+
     var rnames: list((string, string, string)); // tuple (dsetName, item type, id)
 
     for (dsetidx, dsetname) in zip(dsetdom, dsetnames) do {
         for (i, fname) in zip(filedom, filenames) {
             var hadError = false;
             try {
-                (sizes[i], types[dsetidx]) = (getArrSize(fname), getArrType(fname, dsetname));
+                types[dsetidx] = getArrType(fname, dsetname);
+                if types[dsetidx] == ArrowTypes.stringArr {
+                  sizes[i] = getStrColSize(filenames[0], dsetname);
```

Comment:
is there a reason to do `filenames[0]` here instead of `fname`?

Reply:
Nope, that was an error; good catch